app/views/shared/_footer.html.erb

```diff
@@ -0,0 +1,28 @@
+<footer>
+  <div class='wrapper'>
+      <div class='col-1 first'>
+        <h3>Company</h3>
+        <ul>
+          <li><%= link_to "About Us", arts_path %></li>
+          <li><%= link_to "Contact Us", arts_path %></li>
+        </ul>
+      </div>
```

Comment:
Why does this has a `first` class ?
Also a ul of links is a nav so you can change every `u`l with `nav` and remove the `li`s